### PR TITLE
Extend expiration of bounties

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
Allows users to extend expiration date for a bounty

- [x] Use standard modal that exists.
- [x] Add a standard navy blue button and call it 'Extend Expiration'. This button should be placed to the right of Cancel Bounty and before View on Github.
- [x] The Front end UI is applied to look like attached mockups.
- [x] User should be able to select the time frame (leverage what is being used on the funder form, but please adjust the width of the dropdown).
- [x] The ouput of the current time frame + new time frame should display below in an area. This should also display the New Date (August, 13,2018) along with hours.
- [x] Once user clicks Save, a standard confirmation notification should appear.
- [x] The issue details screen should then update to the new expiration date.
- [x] If user clicks cancel, nothing is saved. Please take user back to issue details screen.
- [x] The implement functionality is tested and works.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

bounty details

##### Refers/Fixes

Refs: #1427

### CODE REVIEW

File: app/app/urls.py [hunk 1]
```diff
@@ -184,6 +184,7 @@
 
     # modals
     re_path(r'^modal/get_quickstart_video?', dashboard.views.get_quickstart_video, name='get_quickstart_video'),
+    re_path(r'^modal/extend_issue_deadline?', dashboard.views.extend_issue_deadline, name='extend_issue_deadline'),
```
Comment (mbeacom) [inline app/app/urls.py]: If we're using regex with `?`, you'll want to throw a `/` between `e` and `?`.  The route above it is actually wrong - That means `/modal/get_quickstart_vide` hits the same endpoint as `modal/get_quickstart_video` and `modal/get_quickstart_video/`

File: app/dashboard/templates/extend_issue_deadline.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    {% if user_logged_in %}
+      <div class="col-12">
+        <h5 class="text-center font-body">{% trans "Alpha Seed Implementation (New Feature)" %}</h5>
+        <form class="font-body">
+          <div>
+            <input type="hidden" name="currentExpire" value="{{ bounty.expires_date }}" />
+            <div id="deadline-section" class="pt-2">
+                <label class="form__label" for="expirationTimeDelta">{% trans "Extend by:" %}</label>
+                <div class="form__select2">
+                    <select class="js-select2" name="expirationTimeDelta">
```
Comment (mbeacom) [inline app/dashboard/templates/extend_issue_deadline.html]: Can we adjust the rest of the indentation throughout this file to use 2 space indentation?

**Comment by thelostone-mc** [comment]
@SaptakS added a few more changes on top of it ! 

Current state : 

x1

**Comment by SaptakS** [comment]
I am up for it.

**Comment by mbeacom** [comment]
@PixelantDesign This should be live as of 4 days ago. Are you not seeing it?

**Pull request merged**